index.d.ts

```diff
@@ -113,7 +113,7 @@ export class KafkaConsumer extends Client {
 
     setDefaultConsumeTimeout(timeoutMs: any): void;
 
-    subscribe(topics: string[]): this;
+    subscribe(topics: (string)[]): this;
```

Comment:
`index.d.ts` change still exists. Please make sure only `.npmignore` is listed here.